tools/issue-labeler/src/IssueLabelerService/ConfigurationService.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+
+namespace IssueLabelerService
+{
+    public class ConfigurationService
+    {
+        public IConfiguration _config;
+
+        public ConfigurationService(IConfiguration config) =>
+            _config = config;
+
+        public RepositoryConfiguration GetForRepository(string repository) =>
+            new(_config, repository);
+
+        public RepositoryConfiguration GetDefault() =>
+            new(_config);
+    }
+
+    public class RepositoryConfiguration
```

Comment:
Let's split into two files, please.   .NET convention is to have one class per file, unless the class is embedded.